src/index.js

```diff
@@ -5,9 +5,9 @@ import reportWebVitals from './reportWebVitals';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
+  // <React.StrictMode>
+  <App />,
+  // </React.StrictMode>,
```

Comment:
- [ ] QUite an amazing job! :vulcan_salute: but could you please remove the commented pieces of code on line `8` and `10`? This will prevent your code from being messy, and making it more understandable :wink: